pkg/controller/job/controller_test.go

```diff
@@ -161,7 +161,7 @@ func TestControllerSyncJob(t *testing.T) {
 		"too few active pods, with controller error": {
 			2, 5,
 			fmt.Errorf("Fake error"), 0, 1, 1, 0,
-			0, 0, 1, 1, 0, false,
+			1, 0, 1, 1, 0, false,
```

Comment:
Actually not just patch, I let the fakePodControl takes into account failed [creation](https://github.com/kubernetes/kubernetes/pull/27600/files#diff-44bba2c68a6ea2b4ca0dbb7f4465828eR529) and deletion as well. Sorry I was distracted when typing..